src/components/ListCard/ListCard.tsx

```diff
@@ -13,36 +13,67 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Card, CardActions, CardContent, CardMedia } from '@material-ui/core';
-import React from 'react';
-// import { ItemCardHeader } from '../ItemCardHeader';
+import React, { useMemo } from 'react';
+import { Card, CardActions, CardContent, CardMedia, makeStyles } from '@material-ui/core';
 import {
   Button,
   ItemCardHeader,
   MarkdownContent,
 } from '@backstage/core-components';
+import { BackstageTheme } from '@backstage/theme';
+
+const useStyles = makeStyles<BackstageTheme>(_ => ({
+  linkButton: {
+    backgroundColor: 'transparent',
+    border: 'none',
+    color: '#2E77D0',
```

Comment:
Is there a better way to get the "primary" color (this matches `Button color="primary"`)

Reply:
Figured it out!